WebApi/Controllers/OfficesController.cs

```diff
@@ -21,6 +21,22 @@ public async Task<IActionResult> GetAll()
             return Ok(result);
         }
 
+        [HttpGet("search")]
```

Comment:
it should be same get list request, but with params. we do not need additional endpoint for that

Reply:
Done